Added components for Alerts and Buttons

Anurag021 opened the pull request:
I have added components for Alert and Buttons.
added files alert.css and button.css.
also added alert and button html code in docs.html.
![image](https://user-images.githubusercontent.com/18486358/152121876-b7e8c4b2-ec0d-43c9-8ba0-4edaeb927969.png)
![image](https://user-images.githubusercontent.com/18486358/152122019-7b6811a9-9e33-4a23-abd5-d6b153779c59.png)

partha8 commented on Components/alert.css:
```diff
@@ -0,0 +1,57 @@
+  
+  .components-alert-display{
+
+    border: 0.1rem solid var(--alert-border-color);
```
Vikram mentioned that you can use px when you want to set border size.

partha8 reviewed (commented):
1. Are you using different colors for different components? If so, use only a set of colors, two to four, and have different tints and shades of those colors. Have one primary color which will show the theme of your site. You can refer to other team members' projects. Store these variables in a different CSS file. Your way of using colors is complicated and might result in inconsistency in your design.

2. You can avoid using rems in borders.

3. Rest it looks good

VLeads commented on Components/button.css:
```diff
@@ -1,140 +1,136 @@
+  display: flex;
+  flex-direction: row;
+  padding: 1rem;
+  border: 0.1rem solid black;
```
For border and border radius use px instead of rem.

VLeads reviewed (commented):
Rest code looks good.

VLeads commented:
Also I noticed that you are merging into main. It was asked to merge into dev branch.

Anurag021 commented:
Thanks @partha8  for doing detailed review of the code.. it really helps.
I have implemented all the mentioned changes.
for color part I have added all the color codes in main.css.

Anurag021 commented:
Thanks @VLeads for the review, it helps. 
I have implemented all mentioned changes and also changed the merging branch from main to dev.
